static/js/Story.jsx

```diff
@@ -127,6 +127,46 @@ StorySheetList.propTypes = {
     toggleSignUpModal: PropTypes.func
 };
 
+
+const reviewStateToClassNameMap = {
+    "reviewed": "reviewed",
+    "not reviewed": "notReviewed",
+    "edited": "edited"
+}
+const reviewStateToDisplayedTextMap = {
+    "reviewed": "Reviewed",
+    "not reviewed": "Not Reviewed",
+    "edited": "Edited"
+}
+const ReviewStateIndicator = ({reviewState, onClick}) => {
+    if (!Sefaria.is_moderator || !reviewState){
+        return null}
+    let reviewStateClassName = reviewStateToClassNameMap[reviewState];
+    let displayedText = reviewStateToDisplayedTextMap[reviewState];
+    return (
+    <div className={`button extraSmall reviewState ${reviewStateClassName}`} onClick={onClick}>
+        {displayedText}
+    </div>)
+
+};
+
+const useReviewState = (topic, text) => {
+    let lang = Sefaria.interfaceLang == "english" ? 'en' : 'he';
+    let review = text.descriptions?.[lang]?.review_state;
+    const [reviewState, setReviewState] = useState(review);
```

Comment:
calculate review state for all interface languages and use InterfaceText to display them.